### PR TITLE
Adding chapter 15

### PR DESCRIPTION
**Pull request by OrangeToucan**

### CODE REVIEW

File: docs/15-lexical-blocking.markdown [hunk 1]
```diff
@@ -0,0 +1,381 @@
...
+or
+
+    <GET oblist-name:atom OBLIST>
+
+returns the `OBLIST` whose name is *oblist-name*.
+
+Since there is nothing special about the association of `OBLIST`s and 
+their names, the name of an `OBLIST` can be changed by the use of 
+`PUTPROP`, both on the `OBLIST` and its  name. It is not wise to 
+change the `OBLIST` association without change the name association,
```
Comment (taradinoc) [inline docs/15-lexical-blocking.markdown]: without change -> without changing

File: docs/15-lexical-blocking.markdown [hunk 2]
```diff
@@ -0,0 +1,381 @@
...
+
+or
+
+    <GET oblist-name:atom OBLIST>
+
+returns the `OBLIST` whose name is *oblist-name*.
+
+Since there is nothing special about the association of `OBLIST`s and 
+their names, the name of an `OBLIST` can be changed by the use of 
+`PUTPROP`, both on the `OBLIST` and its  name. It is not wise to
```
Comment (taradinoc) [inline docs/15-lexical-blocking.markdown]: extraneous space between "its" and "name"

File: docs/15-lexical-blocking.markdown [hunk 3]
```diff
@@ -0,0 +1,381 @@
...
+
+Since there is nothing special about the association of `OBLIST`s and 
+their names, the name of an `OBLIST` can be changed by the use of 
+`PUTPROP`, both on the `OBLIST` and its  name. It is not wise to 
+change the `OBLIST` association without change the name association, 
+since you are likely to confused `READ` and `PRINT` terribly.
+
+You can also use `PUT` or `PUTPROP` to remove the association between 
+an `OBLIST` and its name completely. If you want the `OBLIST` to go 
+away (be garbage collect3ed), *and* you want to keep its name around,
```
Comment (taradinoc) [inline docs/15-lexical-blocking.markdown]: typo in "collected"

"and" is underlined in the original, so should be bold (double asterisks) here

File: docs/15-lexical-blocking.markdown [hunk 4]
```diff
@@ -0,0 +1,381 @@
...
+
+    <GET oblist-name:atom OBLIST>
+
+returns the `OBLIST` whose name is *oblist-name*.
+
+Since there is nothing special about the association of `OBLIST`s and 
+their names, the name of an `OBLIST` can be changed by the use of 
+`PUTPROP`, both on the `OBLIST` and its  name. It is not wise to 
+change the `OBLIST` association without change the name association, 
+since you are likely to confused `READ` and `PRINT` terribly.
```
Comment (taradinoc) [inline docs/15-lexical-blocking.markdown]: confused -> confuse

File: docs/15-lexical-blocking.markdown [hunk 5]
```diff
@@ -0,0 +1,381 @@
...
+a `TYPE` name -- points to its `OBLIST`), both of them -- and their 
+association -- will go away without your having to remove the 
+association, of course.) It is not recommended that you remove the 
+name of an `OBLIST` without having it go away, since then `ATOM`s in 
+that `OBLIST` will `PRINT` the name as if they were in no `OBLIST` -- 
+which is defeating the purpose of this whole exercise.
+
+### 15.2.2. MOBLIST
+
+`<MOBLIST atom fix>`
```
Comment (taradinoc) [inline docs/15-lexical-blocking.markdown]: please use indent instead of backtick here

File: docs/15-lexical-blocking.markdown [hunk 6]
```diff
@@ -0,0 +1,381 @@
...
+`ATOM`s, whose name is *atom*, unless there already exists an 
+`OBLIST` of that name, in which case it returns the existing `OBLIST`. 
+*fix* is the size of the `OBLIST` created -- the number of hashing 
+buckets. *fix* is optional (ignored if the `OBLIST` already existed), 
+13 by default. If specified, *fix* should be a prime number, since 
+that allows the hashing to work better.
+
+### 15.2.3. OBLIST?
+
+`<OBLIST? atom>`
```
Comment (taradinoc) [inline docs/15-lexical-blocking.markdown]: please use indent instead of backtick here

File: docs/15-lexical-blocking.markdown [hunk 7]
```diff
@@ -0,0 +1,381 @@
...
+
+### 15.2.2. MOBLIST
+
+`<MOBLIST atom fix>`
+
+("make oblist") creates and returns a new `OBLIST`, containing no 
+`ATOM`s, whose name is *atom*, unless there already exists an 
+`OBLIST` of that name, in which case it returns the existing `OBLIST`. 
+*fix* is the size of the `OBLIST` created -- the number of hashing 
+buckets. *fix* is optional (ignored if the `OBLIST` already existed),
```
Comment (taradinoc) [inline docs/15-lexical-blocking.markdown]: existed -> exists

File: docs/15-lexical-blocking.markdown [hunk 8]
```diff
@@ -0,0 +1,381 @@
...
+*fix* is the size of the `OBLIST` created -- the number of hashing 
+buckets. *fix* is optional (ignored if the `OBLIST` already existed), 
+13 by default. If specified, *fix* should be a prime number, since 
+that allows the hashing to work better.
+
+### 15.2.3. OBLIST?
+
+`<OBLIST? atom>`
+
+returns `#FALSE` `()` if *atom* is not in any `OBLIST`. If *atom* is
```
Comment (taradinoc) [inline docs/15-lexical-blocking.markdown]: one set of backticks around `#FALSE ()` (it's a single MDL expression)

File: docs/15-lexical-blocking.markdown [hunk 9]
```diff
@@ -0,0 +1,381 @@
...
+## 15.3. READ and OBLISTs
+
+`READ` can be explicitly told to look up an `ATOM` in a particular 
+`OBLIST` by giving the `ATOM` a *trailer*. A trailer consists of the 
+characters `!-` (exclamation-point dash) following the `ATOM`, 
+immediately followed by the name of the `OBLIST`. For example,
+
+    A!-OB
+
+specified the unique `ATOM` of `PNAME` `A` which is in the `OBLIST`
```
Comment (taradinoc) [inline docs/15-lexical-blocking.markdown]: specified -> specifies

File: docs/15-lexical-blocking.markdown [hunk 10]
```diff
@@ -0,0 +1,381 @@
...
+
+`<OBLIST? atom>`
+
+returns `#FALSE` `()` if *atom* is not in any `OBLIST`. If *atom* is 
+in an `OBLIST`, it returns that `OBLIST`.
+
+## 15.3. READ and OBLISTs
+
+`READ` can be explicitly told to look up an `ATOM` in a particular 
+`OBLIST` by giving the `ATOM` a *trailer*. A trailer consists of the
```
Comment (taradinoc) [inline docs/15-lexical-blocking.markdown]: double asterisks around **trailer**

File: docs/15-lexical-blocking.markdown [hunk 11]
```diff
@@ -0,0 +1,381 @@
...
+`OBLIST` by giving the `ATOM` a *trailer*. A trailer consists of the 
+characters `!-` (exclamation-point dash) following the `ATOM`, 
+immediately followed by the name of the `OBLIST`. For example,
+
+    A!-OB
+
+specified the unique `ATOM` of `PNAME` `A` which is in the `OBLIST` 
+whose name is the `ATOM` `OB`.
+
+Note that the name of the `OBLIST` must follow the `!-` with *no*
```
Comment (taradinoc) [inline docs/15-lexical-blocking.markdown]: double asterisks around **no**

File: docs/15-lexical-blocking.markdown [hunk 12]
```diff
@@ -0,0 +1,381 @@
...
+characters `!-` (exclamation-point dash) following the `ATOM`, 
+immediately followed by the name of the `OBLIST`. For example,
+
+    A!-OB
+
+specified the unique `ATOM` of `PNAME` `A` which is in the `OBLIST` 
+whose name is the `ATOM` `OB`.
+
+Note that the name of the `OBLIST` must follow the `!-` with *no* 
+separators (like space, tab, carriage-returns, etc.). There is a name
```
Comment (taradinoc) [inline docs/15-lexical-blocking.markdown]: returns -> return

File: docs/15-lexical-blocking.markdown [hunk 13]
```diff
@@ -0,0 +1,381 @@
...
+separators (like space, tab, carriage-returns, etc.). There is a name 
+used by default (section 15.5) which types out and is typed in as 
+`!-`*separator*.
+
+Trailers can be used recursively:
+
+    B!-A!-OB
+
+specified the unique `ATOM` of `PNAME` `B` which is in the `OBLIST` 
+whose name is the unique `ATOM` of `PNAME` `A` which is is in the
```
Comment (taradinoc) [inline docs/15-lexical-blocking.markdown]: is is in -> is in

File: docs/15-lexical-blocking.markdown [hunk 14]
```diff
@@ -0,0 +1,381 @@
...
+
+If an `OBLIST` whose name is given in a trailer does not exist, `READ` 
+creates one, of length 13 buckets.
+
+If trailer notation is not used (the "normal" case), and for an `ATOM` 
+that terminates a trailer, `READ` looks up the `PNAME` of the `ATOM` 
+in a `LIST` of `OBLIST`s, the `LVAL` of the `ATOM` `OBLIST` by 
+default. This looks-up starts with `<1. OBLIST>` and continues until 
+`.OBLIST` is exhausted. If the `ATOM` is not found. `READ` usually 
+inserts it into `<1. OBLIST>`. (It is possible for force `READ` to use
```
Comment (taradinoc) [inline docs/15-lexical-blocking.markdown]: space should be before the dot in `<1 .OBLIST>`

possible for -> possible to

File: docs/15-lexical-blocking.markdown [hunk 15]
```diff
@@ -0,0 +1,381 @@
...
+inserts it into `<1. OBLIST>`. (It is possible for force `READ` to use 
+a different element of the `LIST` of `OBLIST`s for new insertions. If 
+the `ATOM` `DEFAULT` is in that `LIST`, the `OBLIST` following that 
+`ATOM` will be used.)
+
+## 15.4. PRIN1 and OBLISTs
+
+When `PRINT` is given an `ATOM` to output, it outputs as little of the 
+trailer as is necessary to specify the `ATOM` uniquely to `READ`. That 
+is, if the `ATOM` is the *first* `ATOM` of that `PNAME` which `READ`
```
Comment (taradinoc) [inline docs/15-lexical-blocking.markdown]: double asterisks around **first**

File: docs/15-lexical-blocking.markdown [hunk 16]
```diff
@@ -0,0 +1,381 @@
...
+
+Warning: there are obscure cases, which do not occur in normal 
+practice, for which the `PRINT` trailer does not terminate. For 
+instance, if an `ATOM` must have a trailer printed, and the name of 
+the `OBLIST` is an `ATOM` in that very same `OBLIST`, death. Any 
+similar case will also give `PRINT` a hernia.
+
+## 15.5. Initial State
+
+In an initial MDL, `.OBLIST` contains two `OBLIST`s. `<1. OBLIST>`
```
Comment (taradinoc) [inline docs/15-lexical-blocking.markdown]: spacing in `<1 .OBLIST>`

File: docs/15-lexical-blocking.markdown [hunk 17]
```diff
@@ -0,0 +1,381 @@
...
+the `OBLIST` is an `ATOM` in that very same `OBLIST`, death. Any 
+similar case will also give `PRINT` a hernia.
+
+## 15.5. Initial State
+
+In an initial MDL, `.OBLIST` contains two `OBLIST`s. `<1. OBLIST>` 
+initially contains no `ATOM`s, and `<2 .OBLIST>` contains all the 
+`ATOM`s whose `GVAL` are `SUBR`s or `FSUBR`s, as well as `OBLIST`, 
+`DEFAULT`, `T`, etc. It is difficult to lose track of the latter; the 
+specific trailer `!-`*separator* will *always* cause references to
```
Comment (taradinoc) [inline docs/15-lexical-blocking.markdown]: double asterisks around **always**

File: docs/15-lexical-blocking.markdown [hunk 18]
```diff
@@ -0,0 +1,381 @@
...
+similar case will also give `PRINT` a hernia.
+
+## 15.5. Initial State
+
+In an initial MDL, `.OBLIST` contains two `OBLIST`s. `<1. OBLIST>` 
+initially contains no `ATOM`s, and `<2 .OBLIST>` contains all the 
+`ATOM`s whose `GVAL` are `SUBR`s or `FSUBR`s, as well as `OBLIST`, 
+`DEFAULT`, `T`, etc. It is difficult to lose track of the latter; the 
+specific trailer `!-`*separator* will *always* cause references to 
+that `OBLIST`. In addition, the `SUBR ROOT`, which takes no arguments,
```
Comment (taradinoc) [inline docs/15-lexical-blocking.markdown]: separate backticks around `SUBR` and `ROOT`

File: docs/15-lexical-blocking.markdown [hunk 19]
```diff
@@ -0,0 +1,381 @@
...
+
+In an initial MDL, `.OBLIST` contains two `OBLIST`s. `<1. OBLIST>` 
+initially contains no `ATOM`s, and `<2 .OBLIST>` contains all the 
+`ATOM`s whose `GVAL` are `SUBR`s or `FSUBR`s, as well as `OBLIST`, 
+`DEFAULT`, `T`, etc. It is difficult to lose track of the latter; the 
+specific trailer `!-`*separator* will *always* cause references to 
+that `OBLIST`. In addition, the `SUBR ROOT`, which takes no arguments, 
+always returns that `OBLIST`.
+
+The name of `<ROOT>` IS `ROOT`; this `ATOM` is in `<ROOT>` and would
```
Comment (taradinoc) [inline docs/15-lexical-blocking.markdown]: lowercase "is"

File: docs/15-lexical-blocking.markdown [hunk 20]
```diff
@@ -0,0 +1,381 @@
...
+initially contains no `ATOM`s, and `<2 .OBLIST>` contains all the 
+`ATOM`s whose `GVAL` are `SUBR`s or `FSUBR`s, as well as `OBLIST`, 
+`DEFAULT`, `T`, etc. It is difficult to lose track of the latter; the 
+specific trailer `!-`*separator* will *always* cause references to 
+that `OBLIST`. In addition, the `SUBR ROOT`, which takes no arguments, 
+always returns that `OBLIST`.
+
+The name of `<ROOT>` IS `ROOT`; this `ATOM` is in `<ROOT>` and would 
+cause infinite recursion were it not for the use of `!-`*separator*. 
+The name of the initial `<1. OBLIST>` is `INITIAL` (really `INITIAL!-
```
Comment (taradinoc) [inline docs/15-lexical-blocking.markdown]: spacing in `<1 .OBLIST>`

File: docs/15-lexical-blocking.markdown [hunk 21]
```diff
@@ -0,0 +1,381 @@
...
+`MUDDLE!-` is used infrequently by the interpreter when loading 
+compiled programs to fix up references to locations within the 
+interpreter.
+
+The pre-loading of compiled programs may create other `OBLIST`s in an 
+initialized MDL (Lebling, 1979).
+
+## 15.6. BLOCK and ENDBLOCK
+
+These `SUBR`s are analogous to *begin* and *end* in Algol, etc., in
```
Comment (taradinoc) [inline docs/15-lexical-blocking.markdown]: double asterisks around **begin** and **end**

File: docs/15-lexical-blocking.markdown [hunk 22]
```diff
@@ -0,0 +1,381 @@
...
+compiled programs to fix up references to locations within the 
+interpreter.
+
+The pre-loading of compiled programs may create other `OBLIST`s in an 
+initialized MDL (Lebling, 1979).
+
+## 15.6. BLOCK and ENDBLOCK
+
+These `SUBR`s are analogous to *begin* and *end* in Algol, etc., in 
+the way they manipulate static blocking (and in *no* other way.)
```
Comment (taradinoc) [inline docs/15-lexical-blocking.markdown]: double asterisks around **no**

File: docs/15-lexical-blocking.markdown [hunk 23]
```diff
@@ -0,0 +1,381 @@
...
+
+    <BLOCK look-up:list-of-oblists>
+
+returns its argument after "pushing" the current `LVAL` OF THE `ATOM`
```
Comment (taradinoc) [inline docs/15-lexical-blocking.markdown]: lowercase "of the"

File: docs/15-lexical-blocking.markdown [hunk 24]
```diff
@@ -0,0 +1,381 @@
...
+of them, used as stated in section 15.3 to look up `ATOM`s and insert 
+them in `OBLIST`s. If not specified, `.OBLIST` is used. See also 
+section 11.1.1.1, 11.3, and 17.1.3 for other arguments.
+
+### 15.7.2. PARSE and LPARSE (again)
+
+    <PARSE string radix:fix look-up>
+
+as was previously mentioned, applies `READ`'s algorithm to *string* 
+and returns the first MDL object resulting. This *includes* looking up
```
Comment (taradinoc) [inline docs/15-lexical-blocking.markdown]: double asterisks around **includes**

File: docs/15-lexical-blocking.markdown [hunk 25]
```diff
@@ -0,0 +1,381 @@
...
+there is such an `ATOM`; otherwise, it returns `#FALSE ()`. If 
+*string* would `PARSE` into an `ATOM` anyway, `LOOKUP` is faster, 
+although it looks in only one `OBLIST` instead of a `LIST` of them.
+
+### 15.7.4. ATOM
+
+    <ATOM string>
+
+creates and returns a spanking new `ATOM` of `PNAME` *string* which is 
+guaranteed not to be on *any* `OBLIST`.
```
Comment (taradinoc) [inline docs/15-lexical-blocking.markdown]: double asterisks around **any**

File: docs/15-lexical-blocking.markdown [hunk 26]
```diff
@@ -0,0 +1,381 @@
...
+
+An `ATOM` which is not on any `OBLIST` IS `PRINT`ed with a trailer of
```
Comment (taradinoc) [inline docs/15-lexical-blocking.markdown]: lowercase "is"

File: docs/15-lexical-blocking.markdown [hunk 27]
```diff
@@ -0,0 +1,381 @@
...
+### 15.7.5. REMOVE
+
+    <REMOVE string oblist>
+
+removes the `ATOM` of `PNAME` *string* from *oblist* and returns that 
+ATOM. If there is no such `ATOM`, `REMOVE` returns `#FALSE ()`. Also,
+
+    <REMOVE atom>
+
+removes *atom* from its `OBLIST`, if it is one one. It returns *atom*
```
Comment (taradinoc) [inline docs/15-lexical-blocking.markdown]: one one -> on one

File: docs/15-lexical-blocking.markdown [hunk 28]
```diff
@@ -0,0 +1,381 @@
...
+returns it. If there is already an `ATOM` with the same `PNAME` as 
+*atom* in *oblist*, an error occurs. The standard way to avoid the 
+error and always get your *atom* is
+
+    <OR <LOOKUP string oblist> <INSERT string oblist>>
+
+As with `REMOVE`, `INSERT` can also take an `ATOM` as its first 
+argument; this `ATOM` must not be on any `OBLIST` -- it must have been 
+`REMOVE`d, or just created by `ATOM` -- else an error occurs. The 
+`OBLIST` argument is *never* optional. If you would like the new `ATOM`
```
Comment (taradinoc) [inline docs/15-lexical-blocking.markdown]: double asterisks around **never**

File: docs/15-lexical-blocking.markdown [hunk 29]
```diff
@@ -0,0 +1,381 @@
...
+string>` instead.
+
+### 15.7.7. PNAME
+
+    <PNAME atom>
+
+returns a `STRING` (newly created) which is *atom*'s `PNAME` ("printed 
+name"). If trailers are not needed, `PNAME` is much faster than 
+`UNPARSE` on *atom*. (In fact, `UNPARSE` has to go all the way through 
+the `PRINT algorithm *twice*, the first time to see how long a
```
Comment (taradinoc) [inline docs/15-lexical-blocking.markdown]: close the backtick after PRINT

double asterisks around **twice**

File: docs/15-lexical-blocking.markdown [hunk 30]
```diff
@@ -0,0 +1,381 @@
...
+returns a `STRING` (newly created) which is *atom*'s `PNAME` ("printed 
+name"). If trailers are not needed, `PNAME` is much faster than 
+`UNPARSE` on *atom*. (In fact, `UNPARSE` has to go all the way through 
+the `PRINT algorithm *twice*, the first time to see how long a 
+`STRING` is needed.)
+
+### 15.7.8. SPNAME
+
+`SPNAME` ("shared printed name") is identical to `PNAME`, except that 
+the `STRING` is returns shares storage with *atom* (appendix 1), which
```
Comment (taradinoc) [inline docs/15-lexical-blocking.markdown]: is -> it

File: docs/15-lexical-blocking.markdown [hunk 31]
```diff
@@ -0,0 +1,381 @@
...
+name"). If trailers are not needed, `PNAME` is much faster than 
+`UNPARSE` on *atom*. (In fact, `UNPARSE` has to go all the way through 
+the `PRINT algorithm *twice*, the first time to see how long a 
+`STRING` is needed.)
+
+### 15.7.8. SPNAME
+
+`SPNAME` ("shared printed name") is identical to `PNAME`, except that 
+the `STRING` is returns shares storage with *atom* (appendix 1), which 
+is more efficient is the `STRING` will not be modified. `PUT`ting into
```
Comment (taradinoc) [inline docs/15-lexical-blocking.markdown]: is -> if

File: docs/15-lexical-blocking.markdown [hunk 32]
```diff
@@ -0,0 +1,381 @@
...
+
+`SPNAME` ("shared printed name") is identical to `PNAME`, except that 
+the `STRING` is returns shares storage with *atom* (appendix 1), which 
+is more efficient is the `STRING` will not be modified. `PUT`ting into 
+such a `STRING` will cause an error.
+
+## 15.8. Example: Another Solution to the INC Problem
+
+What follows is an example of the way `OBLIST`s are "normally" used to 
+provide "externally available" `ATOM`s and "local `ATOM`s which are
```
Comment (taradinoc) [inline docs/15-lexical-blocking.markdown]: close the quote after local `ATOMs`

File: docs/15-lexical-blocking.markdown [hunk 33]
```diff
@@ -0,0 +1,381 @@
...
+
+This example is rather trivial, but it contains all of the issues, of 
+which there are three.
+
+The first idea is that you should create two `OBLIST`s, one to hold 
+`ATOM`s which are to be known to other users (`INCO`), and the other 
+to hold internal `ATOM`s which are not normally of interest to other 
+(`INCI`). The case above has one `ATOM` in each category.
+
+Second, `INCO` is explicitly used *without* trailers so that
```
Comment (taradinoc) [inline docs/15-lexical-blocking.markdown]: double asterisks around **without**

File: docs/15-lexical-blocking.markdown [hunk 34]
```diff
@@ -0,0 +1,381 @@
...
+
+The first idea is that you should create two `OBLIST`s, one to hold 
+`ATOM`s which are to be known to other users (`INCO`), and the other 
+to hold internal `ATOM`s which are not normally of interest to other 
+(`INCI`). The case above has one `ATOM` in each category.
+
+Second, `INCO` is explicitly used *without* trailers so that 
+surrounding `BLOCK` and `ENDBLOCK`s will have an effect on it. Thus 
+`INCO` will be in the `OBLIST` desired by the user; `INC` will be in 
+`INCO`, and the user can refer to it by saying `INC!-INCO;` `INCI`
```
Comment (taradinoc) [inline docs/15-lexical-blocking.markdown]: semicolon should be outside the backticks

File: docs/15-lexical-blocking.markdown [hunk 35]
```diff
@@ -0,0 +1,381 @@
...
+`ATOM`s which are to be known to other users (`INCO`), and the other 
+to hold internal `ATOM`s which are not normally of interest to other 
+(`INCI`). The case above has one `ATOM` in each category.
+
+Second, `INCO` is explicitly used *without* trailers so that 
+surrounding `BLOCK` and `ENDBLOCK`s will have an effect on it. Thus 
+`INCO` will be in the `OBLIST` desired by the user; `INC` will be in 
+`INCO`, and the user can refer to it by saying `INC!-INCO;` `INCI` 
+will also be in `INCO`, and can be referred to in the same way; 
+finally,  `A` is really `A!-INCI!-INCO. The point of all this is to
```
Comment (taradinoc) [inline docs/15-lexical-blocking.markdown]: close the backtick after A!-INCI!-INCO

File: docs/15-lexical-blocking.markdown [hunk 36]
```diff
@@ -0,0 +1,381 @@
...
+Second, `INCO` is explicitly used *without* trailers so that 
+surrounding `BLOCK` and `ENDBLOCK`s will have an effect on it. Thus 
+`INCO` will be in the `OBLIST` desired by the user; `INC` will be in 
+`INCO`, and the user can refer to it by saying `INC!-INCO;` `INCI` 
+will also be in `INCO`, and can be referred to in the same way; 
+finally,  `A` is really `A!-INCI!-INCO. The point of all this is to 
+structure the nesting of `OBLIST`s.
+
+Finally, if for some reason (like saving storage space) you wish to 
+throw `INCI` away, you can follow the `ENDBLOCK` with:
```
Comment (taradinoc) [inline docs/15-lexical-blocking.markdown]: no colon

**Comment by OrangeToucan** [comment]
Wow, I really did not do a good job here. Sorry about that. The changes have been incorporated. Please re-review.

**Comment by OrangeToucan** [comment]
Thank you for approving this. Not it appears that it needs merging?

**Pull request merged**